scalafix-sbt/src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -20,6 +20,8 @@ object ScalafixPlugin extends AutoPlugin {
       inputKey[Unit]("Run scalafix rule.")
     val scalafixTest: InputKey[Unit] =
       inputKey[Unit]("Run scalafix as a test(without modifying sources).")
+    val scalafixSuppress: InputKey[Unit] =
```

Comment:
PS. I'm not sure there is a need to add an sbt InputTask for this give it can be accomplished with `scalafixCli --suppress`. It might require an update on the sbt-scalafix completions.

Reply:
I have no strong preference. Naming stuff is hard.

Reply:
I'd prefer the longest name since it shouldn't be used often :)